src/modules.json

```diff
@@ -1,50 +1,50 @@
 {
-  "alerts": "",
-  "alertsIssued": "",
-  "alertsActions": "",
-  "alertsFeeds": "",
-  "alertsSources": "",
-  "alertsAssessments": "",
-  "alertsServiceRequests": "",
-  "assessments": "",
-  "assessmentsObservations": "",
-  "assessmentsQuestionnaires": "",
-  "assessmentsQuestions": "",
-  "assessmentsIndicators": "",
-  "emergencyPlans": "",
-  "emergencyPlansPlanner": "",
-  "emergencyPlansActivities": "",
-  "emergencyPlansProcedures": "",
-  "emergencyPlansDisseminations": "",
-  "emergencyPlansActivations": "",
-  "emergencyPlansDrillsExercises": "",
-  "geographicalFeatures": "",
-  "geographicalFeaturesRegions": "",
-  "geographicalFeaturesDistricts": "",
-  "geographicalFeaturesWards": "",
-  "geographicalFeaturesSubwards": "",
-  "geographicalFeaturesEvacuationCenters": "",
-  "geographicalFeaturesFacilities": "",
-  "geographicalFeaturesWarehouses": "",
-  "geographicalFeaturesCriticalInfrastructure": "",
-  "incidents": "",
-  "incidentsCommandCenter": "",
+  "alerts": "Includes an up to date database of ingested emergency alerts from multiple sources in near real time and disseminating them to disaster management stakeholders.",
+  "alertsIssued": "Provides a list of alerts issued from various sources.",
+  "alertsActions": "Contains a list of plan actions that were activated when an alert was issued.",
+  "alertsFeeds": "Provides a means to view actions taken when a certain alert was issued.",
+  "alertsSources": "Provides a list of agencies responsible for issuing various types alerts.",
+  "alertsAssessments": "Provides a way to review and assess the actions taken when an alert is issued.",
```

Comment:
provide a way? .. :smile: which one?... I think this is a list of all assessments(questionnaires and others) conducted after an alert issued